i18n/zh-tw.toml

```diff
@@ -46,21 +46,21 @@ other = "創建文檔議題"
 [post_create_project_issue]
 other = "創建項目議題"
 [post_posts_in]
-other = "張貼在"
+other = "張貼於"
 [post_reading_time]
-other = "分鐘閱讀"
+other = " 分鐘左右可讀完"
 [post_less_than_a_minute_read]
-other = "少於1分鐘"
+other = "1 分鐘內可讀完"
```

Comment:
Based on my research, the original prompt is better here, please revert this change or explain why your new text is better.

Reply:
@chalin As a matter of fact, in Chinese (no matter Simplified or Traditional), "less than a mimute" is quite a broken phrase or sentence. In my opinion, this one and the one above read much better in both Simplified and Traditional Chinese

Reply:
Thank you for the feedback. Ok, let's go with your update then.